examples/loop_optimizations_service/benchmarks/BUILD

```
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+filegroup(
+    name = "benchmarks",
+    srcs = glob(["*.c"]) + [
+        "//compiler_gym/third_party/neuro-vectorizer:header",
+    ],
```

Nitpick: no need to glob here:

```suggestion
    srcs = [
        "add.c",
        "conv2d.c",
        "offsets1.c",
        "//compiler_gym/third_party/neuro-vectorizer:header",
    ]
```

---

Since this is a benchmarks directory, I had in mind that in the future a lot of .c files will be added, so glob will make things easier. 

But I just looked in the BUILD of other benchmark directories and found that we don't use glob. Will remove it now